development/prepare-code-for-review.md

```diff
@@ -12,7 +12,13 @@ If it's not obvious, it can also be helpful to include instructions on how to te
 
 ## Review Your Own Branch
 
-When you have your initial version for the proposed change working, look at your changes once again and ask yourself if anything could be improved before marking it "Ready for Review".
+When you have your initial version for the proposed change working, look at your changes once again and ask yourself if anything could be improved before marking it "Ready for Review". Be sure to check that:
+
+✅️ Your branch adheres to the elementary [code style guide](https://docs.elementary.io/develop/writing-apps/code-style)
+
+✅️ No New Terminal warnings are introduced
+
+✅️ Your branch passes CI
```

Comment:
Is the abbreviation "CI" explained anywhere (before this page is read)?  If not, more explanation is required or a link to an explanation.

Reply:
Updated this to "status checks" to be consistent with the language GitHub uses and linked to their docs